terraform/cleanup.sh

```diff
@@ -104,6 +104,10 @@ gcloud scheduler jobs delete nightly-builds \
     -q \
     || true
 
+# Cloud Storage buckets (for session data)
+gsutil rm -r "gs://${PROD_PROJECT}-sessions" || true
```

Comment:
Once the bucket is deleted, re-creating it is not trivial. We either need to create it with a random name or we should reuse it. Perhaps the test runner should import the bucket state before apply, instead of remove?